content/blog/2023/web-pwa-native-PWA3.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Part 3 - Responsive Web App vs PWA vs Native Mobile App
+sub: When to choose a progressive web app (PWA) for you digital product
+author: sondra
+date: 2023-07-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part3-progressive.jpg
+  alt: |
+    Pen and notebook with a sketch of a website interface
+summary: |
+  If you're weighing the performance optimazation
+  and device integration opportunities
+  of a native mobile app
+  against the broad reach and lower cost
+  of the responsive web app,
```

Comment:
Should this be "a" since the previous phrase used "a" when it said "of a native mobile app"
```suggestion
  of a responsive web app,
```